recipes/pylsp-rope/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
You don't need to include this copy of the license file since it's included in the source distribution. Please delete it.